.travis/linux_openresty_runner.sh

```diff
@@ -94,6 +94,13 @@ do_install() {
         tar -xvf grpcurl-amd64.tar.gz
         mv grpcurl build-cache/
     fi
+
+    if [ ! -f "build-cache/eureka" ]; then
+        wget https://github.com/qiujiayu/apisix-eureka/releases/download/v1.9.8/eureka.tar.gz
```

Comment:
not download eureka from thrid-party

Reply:
will move to API7.

Reply:
eureka not has his own download website?

Reply:
no, just a jar cannot be executed. need build by ourselves

Reply:
the download url be changed.

Reply:
maybe offical docker image is good for us.  we just need the simple service , avoid personal config on the eureka.